clusterloader2/testing/density/config.yaml

```diff
@@ -0,0 +1,116 @@
+automanagedNamespaces: 3
+steps:
+- measurements:
+  - Identifier: TimerGlobal
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: TimerCreate
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: APICallLatency
+    Method: APICallLatency
+    Params:
+      action: reset
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 1
+    tuningSet: TuningSet100
+    objectBundle:
+    - basename: rc1
```

Comment:
Again here I suggest giving a more meaningful name to what kind of rc this is. For e.g `saturationRC` (as we create these RCs to saturate the cluster with many many pods).